tika-parsers/src/test/java/org/apache/tika/parser/ocr/TesseractOCRConfigTest.java

```diff
@@ -70,6 +81,11 @@ public void testFullConfig() throws Exception {
         assertEquals("Invalid overridden minFileSizeToOcr value", 1, config.getMinFileSizeToOcr());
         assertEquals("Invalid overridden maxFileSizeToOcr value", 2000000, config.getMaxFileSizeToOcr());
         assertEquals("Invalid overridden timeout value", 240, config.getTimeout());
+        assertEquals("Invalid overridden ImageMagickPath value", "/usr/local/bin/", config.getImageMagickPath());
```

Comment:
@Zarana-Parekh can you add a check using Apache commons-util to make sure this is a UNIX OS before running the above test.